proxmoxtf/resource/container.go

```diff
@@ -67,6 +67,7 @@ const (
 	dvResourceVirtualEnvironmentContainerStarted                           = true
 	dvResourceVirtualEnvironmentContainerTemplate                          = false
 	dvResourceVirtualEnvironmentContainerUnprivileged                      = false
+	dvResourceVirtualEnvironmentContainerStartOnBoot                       = false
```

Comment:
Shouldn't it be `true` by default, to be backward compatible with the old code / existing deployments?

Reply:
Minor, I would put all new *StartOnBoot code right beside the *Started code (constants, schema block, assignments in code, etc)

Reply:
> Shouldn't it be `true` by default, to be backward compatible with the old code / existing deployments?

Yep you're right. I initially set the default value to false since it seems to be the default in the UI but in order to be backward compatible, I guess true is better.

> Minor, I would put all new *StartOnBoot code right beside the *Started code (constants, schema block, assignments in code, etc)

Done!